### PR TITLE
Remove draft banner from main dashboard

### PR DESCRIPTION
**Pull request by cruddasj**
This was missed in the previous round - so updated via PR.

### CODE REVIEW

File: app/views/dashboard/user_dashboard.html.haml [hunk 1]
```diff
@@ -2,6 +2,8 @@
   .govuk-grid-row
     .govuk-grid-column-two-thirds
       %h1.govuk-heading-l Your fleet account
+      %p
```
Comment (likeuwill) [inline app/views/dashboard/user_dashboard.html.haml]: `%p On this page you can upload your vehicles and make a payment`

File: app/views/dashboard/user_dashboard.html.haml [hunk 2]
```diff
@@ -2,6 +2,8 @@
   .govuk-grid-row
     .govuk-grid-column-two-thirds
       %h1.govuk-heading-l Your fleet account
+      %p 
+      On this page you can upload your vehicles and make a payment
       %p On this page you can upload your vehicles and make a payment
```
Comment (likeuwill) [inline app/views/dashboard/user_dashboard.html.haml]: duplicate?

Comment (cruddasj) [reply app/views/dashboard/user_dashboard.html.haml]: Ah - good spot - merge conflict :/

**Comment by cruddasj** [comment]
Note the updated copy is included from the prototype and the service name has been updated based on discussion with the Product Owner.

**Pull request merged**